packages/config/config/devices/0x027a/zen31.json

```
@@ -325,146 +208,114 @@
 		},
 		{
 			"#": "40[0x01]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
 			"label": "Scene Control for IN1. click/toggle 1 time to trigger",
 			"description": "Click/toggle the connected switch 1 time to trigger a scene.",
```

These labels/descriptions don't match the style guide. In the Fibaro files we have these labels that could be adopted here:
```suggestion
			"label": "Input 1: Send Scenes (Key Pressed 1x)"
```

---

https://github.com/zwave-js/node-zwave-js/pull/4619/commits/504ab2e122c1ad4e5262500a7d4ffb87134a87e1